automated/deployment/lava_prog_device.sh

```diff
@@ -22,9 +22,15 @@ TEST_NAME="image_deployment"
 # find image and extract
 cd /lava-lxc || exit
 
-ARCHIVE=$(find . -iname "*tar*")
-tar xzvf "$ARCHIVE"
-rm "$ARCHIVE"
+
+if ! command -v unxz &> /dev/null; then
+    echo "unxz not intalled."
+    apt-get update
+    apt-get install -y xz-utils
+fi
+
+ARCHIVE=$(find . -iname "*xz*")
```

Comment:
The depth and type of file of the `find` command would be useful here as we are only looking for a file in the current directory.
The image should also always end with a `.xz` suffix.

```suggestion
ARCHIVE=$(find . -maxdepth 1 -iname "*.xz" -type f)
```

Reply:
This is not resolved, line 32 still contains the inaccurate `find` command. Instead, it seems to have snuck in at line 35 where it's wrong.